ansible/roles/test/files/ptftests/sad_path.py

```diff
@@ -62,78 +63,91 @@ def cmd(self, cmds):
 
     def select_vm(self):
         self.vm_list.sort()
-        # use the day of the month to select a VM from the list for the sad pass operation
-        vm_index = datetime.datetime.now().day % len(self.vm_list)
-        self.neigh_vm = self.vm_list.pop(vm_index)
+        vm_len = len(self.vm_list)
+        # use the day of the month to select start VM from the list for the sad pass operation
+        # neigh_vm list will contain cnt number of VMs starting from the start VM. vm_list will have the rest of the VMs
+        vm_index = datetime.datetime.now().day % vm_len
```

Comment:
>vm_len [](start = 49, length = 6)

How to prevent div by 0?

Reply:
will do the mod operation only if vm_list is non empty otherwise I will set it to 0